custom_components/london_tfl/sensor.py

```diff
@@ -206,3 +210,9 @@ def extra_state_attributes(self):
         attributes['data'] = data
 
         return attributes
+
+    def shortenPlatformName(destinationName, shorten):
+        if CONF_SHORTEN_STATION_NAMES:

```

Comment:
This is wrong. It should be `if self.shortenStationNames`. 
`CONF_SHORTEN_STATION_NAMES` always evaluates to true.